templates/monitrc.erb

```diff
@@ -32,6 +32,9 @@ set httpd port <%= @httpd_port %> and
    <%- if !@httpd_user.empty? && !@httpd_password.empty? -%>
    allow <%= @httpd_user %>:<%= @httpd_password %>
    <%- end -%>
+   <%- @httpd_groups.each do |user| -%>
```

Comment:
Could you tell me why you used the name "user" for the items of "groups"?
In the Monit doc (https://mmonit.com/monit/documentation/monit.html#PAM), it is called "group".
Can I use this instead?